README.md

```diff
@@ -155,4 +155,4 @@ man ceph-salt
 
 [ceph]: https://ceph.io/
 [salt]: https://www.saltstack.com/
-[cephadm]: https://docs.ceph.com/docs/master/mgr/cephadm/
+[cephadm]: https://docs.ceph.com/en/octopus/cephadm/
```

Comment:
The idea is to change readthedocs to make https://docs.ceph.com/en/latest/cephadm/ point to the latest *stable* release. What do you think of pointing to https://docs.ceph.com/en/latest/cephadm/ ?. This would fix the 404

Reply:
@sebastian-philipp If ceph-salt been tested on the latest cephadm, then I have no objections.

I have used ceph-salt only with octopus cephadm. That's why I wrote "octopus" instead of "latest", here.

Reply:
> This would fix the 404

My patch also fixes the 404, no?

Reply:
My concern is that the latest cephadm might differ semantically from the cephadm ceph-salt is currently supporting.

Once ceph-salt is confirmed to support a more recent cephadm (e.g. pacific), we can change this link to point to it.

But to have it just point to whatever is the latest cephadm is misleading IMO.